src/levanter/data/text.py

```diff
@@ -524,13 +524,18 @@ def fsspec_expand_glob(url):
         return urls
 
 
+@dataclass
+class LMDatasetMixtureComponentConfig(LMDatasetSourceConfig):
```

Comment:
The only thing different between the base and derived class is cache_dir?  Seems weird since this on the surface doesn't seem to relate to mixtures...

Reply:
yeah i can just put it in the base i guess